### PR TITLE
Update Topology introduction and viewing sections to mention 2 views,…

### PR DESCRIPTION
**Pull request by rishumehra**
[RHDEVDOCS-2329](https://issues.redhat.com/browse/RHDEVDOCS-2329): Update Topology introduction and viewing sections to mention 2 views, list, and graph.

This update is for OpenShift 4.7

### CODE REVIEW

File: modules/odc-interacting-with-applications-and-components.adoc [hunk 1]
```diff
@@ -5,21 +5,21 @@
...
+** Hover your cursor over the lower left icon on the Pod to see the name of the latest build and its status. The status of the application build is indicated as *New* (image:odc_build_new.png[title="New Build"]), *Pending* (image:odc_build_pending.png[title="Pending Build"]), *Running* (image:odc_build_running.png[title="Running Build"]), *Completed* (image:odc_build_completed.png[title="Completed Build"]), *Failed* (image:odc_build_failed.png[title="Failed Build"]), and *Canceled* (image:odc_build_canceled.png[title="Canceled Build"]).
+* Use the *View Shortcuts* menu listed on the upper-right of the screen to navigate components in the *Topology view*.
+* Use the *Find by name* field to select the components with component names that match the query. Search results may appear outside of the visible area; click *Fit to Screen* from the lower-left toolbar to re-size the *Topology view* to show all components.
+* Use the *Display Options* drop-down list to configure the *Topology view* of the various application groupings. The options are available depending on the types of components deployed in the project:
```
Comment (andrewballantyne) [inline modules/odc-interacting-with-applications-and-components.adoc:22]: Hey, I noticed there are other features of the Display Options drop-down we do not mention. Is this by intention or accidental omission?

The modes and the expand/collapse features are what I am referring to:
image

File: modules/odc-interacting-with-applications-and-components.adoc [hunk 2]
```diff
@@ -5,21 +5,21 @@
 [id="odc-interacting-with-applications-and-components_{context}"]
 = Interacting with the application and the components
 
-The *Topology* view in the *Developer* perspective of the web console provides the following options to interact with the application and the components:
+By default, the *Topology view* renders in the *Developer* perspective of the web console provides the following options to interact with the application and the components:
 
-* Click *Open URL* (image:odc_open_url.png[title="Application Link"]) to see your application exposed by the route on a public URL.
-* Click *Edit Source code* to access your source code and modify it.
+* Use the *List View* icon to see a list of all your applications and use the *Topology View* icon to switch back to the *Topology* view.
+** Click *Open URL* (image:odc_open_url.png[title="Application Link"]) to see your application exposed by the route on a public URL.
```
Comment (Preeticp) [inline modules/odc-interacting-with-applications-and-components.adoc:11]: On top of this bullet point you can add: The *Topology view* provides the following options to interact with the application and the components:
and sorry, but can we move the bullet points back to level 1.

File: modules/odc-interacting-with-applications-and-components.adoc [hunk 3]
```diff
@@ -5,21 +5,21 @@
...
-* Use the *Find by name* field to select the components with component names that match the query. Search results may appear outside of the visible area; click *Fit to Screen* from the lower-left toolbar to resize the *Topology* view to show all components.
-* Use the *Display Options* drop-down list to configure the *Topology* view of the various application groupings. The options are available depending on the types of components deployed in the project:
+** Hover your cursor over the lower left icon on the Pod to see the name of the latest build and its status. The status of the application build is indicated as *New* (image:odc_build_new.png[title="New Build"]), *Pending* (image:odc_build_pending.png[title="Pending Build"]), *Running* (image:odc_build_running.png[title="Running Build"]), *Completed* (image:odc_build_completed.png[title="Completed Build"]), *Failed* (image:odc_build_failed.png[title="Failed Build"]), and *Canceled* (image:odc_build_canceled.png[title="Canceled Build"]).
+* Use the *View Shortcuts* menu listed on the upper-right of the screen to navigate components in the *Topology view*.
```
Comment (Preeticp) [inline modules/odc-interacting-with-applications-and-components.adoc:20]: let's move the bullet points from here to the start before we list the topology view bullet points.

File: modules/odc-interacting-with-applications-and-components.adoc (hunk 1 above)
Comment (Preeticp) [reply modules/odc-interacting-with-applications-and-components.adoc:22]: @rishumehra let's add these too, please.

File: modules/odc-viewing-application-topology.adoc [hunk 4]
```diff
@@ -5,7 +5,7 @@
 [id="odc-viewing-application-topology_{context}"]
 = Viewing the topology of your application
 
-You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Topology* view where you can see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
+You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Topology view* where you can switch to *List view* also and see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
```
Comment (Preeticp) [inline modules/odc-viewing-application-topology.adoc:8]: Suggest adding the sentence at the end of the paragraph: 
If required, you can also switch to the List View to monitor your applications.

**Review by Preeticp — CHANGES_REQUESTED** [review]

File: modules/odc-viewing-application-topology.adoc (hunk 4 above)
Comment (rishumehra) [reply modules/odc-viewing-application-topology.adoc:8]: Done. Incorporated the changes.

File: modules/odc-viewing-application-topology.adoc [hunk 5]
```diff
@@ -7,27 +7,21 @@
...
+** *Mode* (*Connectivity* or *Consumption*)
+** *Expand* group
+*** Virtual Machines: Toggle to show or hide the virtual machines.
+*** Application Groupings: Clear to condense the application groups into cards with an overview of an application group and alerts associated with it.
+*** Helm Releases: Clear to condense the components deployed as Helm Release into cards with an overview of a given release.
+*** Knative Services: Clear to condense the Knative Service components into cards with an overview of a given component.
+*** Operator Groupings: Clear to condense the components deployed with an Operator into cards with an overview of the given group.
+** *Show* elements based on *Pod Count* or *Labels*
+*** Pod Count: Select to show the number of pods of a component in the component icon.
+*** Labels: Toggle to show or hide the component labels.
```
Comment (rishumehra) [inline modules/odc-viewing-application-topology.adoc:26]: Made changes as per the *Display Options* menu.

**Review by rishumehra — COMMENTED** [review]
Updated odc-viewing-application-topology.adoc only in this commit.

**Review by rishumehra — COMMENTED** [review]
Updated odc-interacting-with-applications-and-components.adoc only in this commit.

**Review by gajanan-more — APPROVED** [review]

File: modules/odc-interacting-with-applications-and-components.adoc [hunk 6]
```diff
@@ -5,8 +5,9 @@
 [id="odc-interacting-with-applications-and-components_{context}"]
 = Interacting with the application and the components
 
-The *Topology* view in the *Developer* perspective of the web console provides the following options to interact with the application and the components:
+The *Topology View* in the Developer perspective of the web console provides the following options to interact with the application and the components:
 
+* Use the *Shrtcuts* menu listed on the upper-right of the screen to navigate components in the *Topology* view.
```
Comment (Preeticp) [inline modules/odc-interacting-with-applications-and-components.adoc:10]: *Shortcuts*
there is no need for the word 'listed'
upper right - see ISG pg206, chpt 7

File: modules/odc-viewing-application-topology.adoc [hunk 7]
```diff
@@ -5,29 +5,23 @@
 [id="odc-viewing-application-topology_{context}"]
 = Viewing the topology of your application
 
-You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Topology* view where you can see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
+You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Topology view* where you can switch to *List view* also and see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
```
Comment (Preeticp) [inline modules/odc-viewing-application-topology.adoc:8]: Post the new changes I think we can revert this section to the original way: 
...you are directed automatically to the *Topology* view where you can see the status of the...

File: modules/odc-viewing-application-topology.adoc [hunk 8]
```diff
@@ -5,29 +5,23 @@
...
-For more information about pod status, see the link:https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase[Kubernetes documentation].
+You can customize the views as required using the following:
 
-After you create an application and an image is deployed, the status is shown as *Pending*. After the application is built, it is displayed as *Running*.
+* Use the *Find by name* field to select the components with component names that match the query. Search results may appear outside of the visible area; click *Fit to Screen* from the lower-left toolbar to resize the *Topology* view to show all components.
```
Comment (Preeticp) [inline modules/odc-viewing-application-topology.adoc:14]: to select the components with component names that match the query.
How about 'to find the required components.'

File: modules/odc-viewing-application-topology.adoc [hunk 9]
```diff
@@ -5,29 +5,23 @@
...
+* Use the *Display Options* drop-down list to configure the *Topology* view of the various application groupings. The options are available depending on the types of components deployed in the project:
 
-.Application topology
-image::odc_application_topology.png[]
+** *Mode* (*Connectivity* or *Consumption*)
```
Comment (Preeticp) [inline modules/odc-viewing-application-topology.adoc:17]: Please check with the team and expand this. 
Connectivity and Consumption would need to be individual nested bullets like Virtual Machines etc. and add a small description for both please.

**Review by Preeticp — CHANGES_REQUESTED** [review]
@rishumehra overall the reorg of the content looks good and makes sense.
I have a left a few comments, please take care of them and we should be good.

File: modules/odc-interacting-with-applications-and-components.adoc (hunk 6 above)
Comment (rishumehra) [reply modules/odc-interacting-with-applications-and-components.adoc:10]: Fixed

File: modules/odc-viewing-application-topology.adoc (hunk 7 above)
Comment (rishumehra) [reply modules/odc-viewing-application-topology.adoc:8]: Fixed

File: modules/odc-viewing-application-topology.adoc (hunk 8 above)
Comment (rishumehra) [reply modules/odc-viewing-application-topology.adoc:14]: Fixed

File: modules/odc-viewing-application-topology.adoc (hunk 9 above)
Comment (rishumehra) [reply modules/odc-viewing-application-topology.adoc:17]: Updating the information in the commit as suggested.

File: modules/odc-interacting-with-applications-and-components.adoc [hunk 10]
```diff
@@ -5,8 +5,9 @@
 [id="odc-interacting-with-applications-and-components_{context}"]
 = Interacting with the application and the components
```
Comment (Preeticp) [inline modules/odc-interacting-with-applications-and-components.adoc:6]: This is not part of this PR, but please change the title to 'Interacting with applications and components' since this topic is about generic applications and not one particular application.

File: modules/odc-interacting-with-applications-and-components.adoc [hunk 11]
```diff
@@ -5,8 +5,9 @@
...
 
-The *Topology* view in the *Developer* perspective of the web console provides the following options to interact with the application and the components:
+The *Topology View* in the Developer perspective of the web console provides the following options to interact with the application and the components:
```
Comment (Preeticp) [inline modules/odc-interacting-with-applications-and-components.adoc:8]: s/with the application and the components/with applications and components

File: modules/odc-viewing-application-topology.adoc [hunk 12]
```diff
@@ -5,29 +5,24 @@
...
+The *Topology* view also provides you the option to monitor your applications using the *List* view. Use the *List View* icon to see a list of all your applications and use the *Topology View* icon to switch back to the graph view.
+
+You can customize the views as required using the following:
+
+* Use the *Find by name* field to find the required components. Search results may appear outside of the visible area; click *Fit to Screen* from the lower-left toolbar to resize the *Topology* view to show all components.
+* Use the *Display Options* drop-down list to configure the *Topology* view of the various application groupings. The options are available depending on the types of components deployed in the project:
+
+** *Mode* (*Connectivity* or *Consumption*)
+*** Connectivity: to show all the connections between different nodes in the  topology.
+*** Consumption: to shows all nodes in consumption mode without any connection in the topology.
```
Comment (Preeticp) [inline modules/odc-viewing-application-topology.adoc:19]: Suggest:
Connectivity: Select to show all the connections between the different nodes in the topology.
Consumption: Select to show the resource consumption for all nodes in the topology. 
(please verify with the developers that they are displaying only the resource consumption)

File: modules/odc-viewing-application-topology.adoc [hunk 13]
```diff
@@ -5,29 +5,24 @@
...
-* *J*: `Job`
-* *P*: `Pod`
-* *SS*: `StatefulSet`
-* image:odc_serverless_app.png[title="Serverless Application"] (Knative): A serverless application
-+
-[NOTE]
-====
-Serverless applications take some time to load and display on the *Topology* view. When you create a serverless application, it first creates a service resource and then a revision. After that, it is deployed and displayed on the *Topology* view. If it is the only workload, you might be redirected to the *Add* page. After the revision is deployed, the serverless application is displayed on the *Topology* view.
-====
+You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Topology view* where you are directed automatically to the Topology view where you can see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
```
Comment (Preeticp) [inline modules/odc-viewing-application-topology.adoc:8]: Remove 'where you are directed automatically to the Topology view', it is repeated.

**Review by Preeticp — COMMENTED** [review]
Looks good overall. A few nits and we should be good to push it to the OCP team.

File: modules/odc-interacting-with-applications-and-components.adoc (hunk 10 above)
Comment (rishumehra) [reply modules/odc-interacting-with-applications-and-components.adoc]: Done

File: modules/odc-interacting-with-applications-and-components.adoc (hunk 11 above)
Comment (rishumehra) [reply modules/odc-interacting-with-applications-and-components.adoc]: Done

File: modules/odc-viewing-application-topology.adoc (hunk 12 above)
Comment (rishumehra) [reply modules/odc-viewing-application-topology.adoc]: Done

File: modules/odc-viewing-application-topology.adoc (hunk 13 above)
Comment (rishumehra) [reply modules/odc-viewing-application-topology.adoc]: Done

File: modules/odc-interacting-with-applications-and-components.adoc [hunk 14]
```diff
@@ -3,10 +3,11 @@
 // applications/application_life_cycle_management/odc-viewing-application-composition-using-topology-view.adoc
 
 [id="odc-interacting-with-applications-and-components_{context}"]
-= Interacting with the application and the components
+= Interacting with applications and components
 
-The *Topology* view in the *Developer* perspective of the web console provides the following options to interact with the application and the components:
+The *Topology View* in the *Developer* perspective of the web console provides the following options to interact with applications and components:
```
Comment (bergerhoffer) [inline modules/odc-interacting-with-applications-and-components.adoc:8]: This made the change from `*Topology* view` to `*Topology View*`, but the addition on line 10 uses it the old way of `*Topology* view`. Just wondering if it needs to be consistent throughout, or if "**Topology View**" is the exact term in the navigation panel, and "**Topology** view" is what is used when speaking of it generally?

File: modules/odc-interacting-with-applications-and-components.adoc [hunk 15]
```diff
@@ -16,15 +17,25 @@ This feature is available only when you create applications using the *From Git*
...
-** *Labels*: Toggle to show or hide the component labels.
-** *Application Groupings*: Clear to condense the application groups into cards with an overview of an application group and alerts associated with it.
-** *Helm Releases*: Clear to condense the components deployed as Helm Release into cards with an overview of a given release.
-** *Knative Services*: Clear to condense the Knative Service components into cards with an overview of a given component.
-** *Operator Groupings* Clear to condense the components deployed with an Operator into cards with an overview of the given group.
+* The status or phase of the Pod is indicated by different colors and tooltips as *Running* (image:odc_pod_running.png[title="Pod Running"]), *Not Ready* (image:odc_pod_not_ready.png[title="Pod Not Ready"]), *Warning*(image:odc_pod_warning.png[title="Pod Warning"]), *Failed*(image:odc_pod_failed.png[title="Pod Failed"]), *Pending*(image:odc_pod_pending.png[title="Pod Pending"]), *Succeeded*(image:odc_pod_succeeded.png[title="Pod Succeeded"]), *Terminating*(image:odc_pod_terminating.png[title="Pod Terminating"]), or *Unknown*(image:odc_pod_unknown.png[title="Pod Unknown"]).
```
Comment (bergerhoffer) [inline modules/odc-interacting-with-applications-and-components.adoc:20]: phase of the Pod -> phase of the pod

File: modules/odc-viewing-application-topology.adoc [hunk 16]
```diff
@@ -5,29 +5,24 @@
...
-* *J*: `Job`
-* *P*: `Pod`
-* *SS*: `StatefulSet`
-* image:odc_serverless_app.png[title="Serverless Application"] (Knative): A serverless application
-+
-[NOTE]
-====
-Serverless applications take some time to load and display on the *Topology* view. When you create a serverless application, it first creates a service resource and then a revision. After that, it is deployed and displayed on the *Topology* view. If it is the only workload, you might be redirected to the *Add* page. After the revision is deployed, the serverless application is displayed on the *Topology* view.
-====
+You can navigate to the *Topology* view using the left navigation panel in the *Developer* perspective. After you create an application, you are directed automatically to the *Topology view* where you can see the status of the application pods, quickly access the application on a public URL, access the source code to modify it, and see the status of your last build. You can zoom in and out to see more details for a particular application.
```
Comment (bergerhoffer) [inline modules/odc-viewing-application-topology.adoc:8]: Again, I see several different version of bolding just Topology, or both Topology and view, and in the preview module, View was capitalized when they were both bolded. Just need some more consistency here.

**Review by bergerhoffer — COMMENTED** [review]
Just a few minor comments, otherwise looks good!

File: modules/odc-viewing-application-topology.adoc [hunk 17]
```diff
@@ -5,29 +5,24 @@
...
+
+The *Topology* view also provides you the option to monitor your applications using the *List* view. Use the *List View* icon to see a list of all your applications and use the *Topology View* icon to switch back to the graph view.
```
Comment (Preeticp) [inline modules/odc-viewing-application-topology.adoc:10]: s/monitor your applications using the *List* view./monitor your applications using the list view.
*List View*/*List view* view is in small case on the UI. Keep the names of GUI elements identical to the way it is displayed on the GUI.
*Topology View*/ *Topology view*

File: modules/odc-viewing-application-topology.adoc (hunk 16 above)
Comment (Preeticp) [reply modules/odc-viewing-application-topology.adoc:8]: Hi Andrea, adding some context: We realized this was going to cause confusion so we have initiated a conversation with the dev and UX teams. On the left navigation panel one of the options we have is Topology, clicking on this shows the Topology view for apps. This view has 2 options List view and Topology view. So we are in a weird situation where we refer to **Topology** view and **Topology view** as per the context. As per the devs, *Topology view* should be *Graph view* and we will update it as soon as we see this change in.
Screenshot from 2021-01-19 14-18-05

File: modules/odc-interacting-with-applications-and-components.adoc (hunk 14 above)
Comment (Preeticp) [reply modules/odc-interacting-with-applications-and-components.adoc:8]: It must be **Topology view** on both Line 8 and 10

Comment (rishumehra) [reply modules/odc-interacting-with-applications-and-components.adoc:8]: Will make the change in the new commit.

File: modules/odc-interacting-with-applications-and-components.adoc (hunk 15 above)
Comment (rishumehra) [reply modules/odc-interacting-with-applications-and-components.adoc:20]: Will make this change.

File: modules/odc-viewing-application-topology.adoc (hunk 17 above)
Comment (rishumehra) [reply modules/odc-viewing-application-topology.adoc:10]: Will incorporate the changes suggested in the new commit.

File: modules/odc-viewing-application-topology.adoc (hunk 9 above)
Comment (rishumehra) [reply modules/odc-viewing-application-topology.adoc:17]: Done

**Comment by rishumehra** [comment]
Closing this PR and raising another one on behalf of this.

File: modules/odc-viewing-application-topology.adoc (hunk 16 above)
Comment (bergerhoffer) [reply modules/odc-viewing-application-topology.adoc]: Thanks for the explanation! I think my main concern this time was that "View" was also capitalized. But it looks like that was taken care of in the new PR, so we are good to go!
